pkg/operator/audit/audit.go

```diff
@@ -0,0 +1,104 @@
+package audit
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	kyaml "k8s.io/apimachinery/pkg/util/yaml"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
+
+	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/v311_00_assets"
+	libgoapiserver "github.com/openshift/library-go/pkg/operator/configobserver/apiserver"
+)
+
+const (
+	policyAsset = "v3.11.0/openshift-apiserver/audit-policies-cm.yaml"
+)
+
+func NewAuditPolicyPathGetter() (libgoapiserver.AuditPolicyPathGetterFunc, error) {
+	policies, err := readPolicyNamesFromResource()
+	if err != nil {
+		return nil, err
+	}
+
+	return func(profile string) (string, error) {
+		const (
+			auditPolicyFilePath = "/etc/kubernetes/static-pod-resources/configmaps/openshift-apiserver-audit-policies"
```

Comment:
how it will end up in that path?

Reply:
- use the static resource controller that creates the ConfigMap from the `bindata`
- add the configmap to the revision controller.
- mount the revision enabled ConfigMap to the pod.
```
        volumeMounts:
        - mountPath: /var/run/configmaps/openshift-apiserver-audit-policies
          name: openshift-apiserver-audit-policies
```

```
volumes:
      - name: openshift-apiserver-audit-policies
        configMap:
          name: openshift-apiserver-audit-policies-${REVISION}
```